src/core/udb-api.service.js

```diff
@@ -422,12 +422,14 @@ function UdbApi(
   };
 
   /**
-   * @param {URL} eventId
+   * @param {URL} id
+   * @param {string} type
    * @return {*}
    */
-  this.getHistory = function (eventId) {
+  this.getHistory = function (id, type) {
+    var url = (type === 'place') ? appConfig.baseUrl + 'places/' + id : id;
```

Comment:
Hmm, I see the need for this workaround, but could we maybe make it consistent so that we always pass a uuid as `id` and not a url in the case of events?
That means we will need to do something similar to `var placeId =  $scope.placeId.split('/').pop()` for events, but at least then there can be no confusion about what kind of string `id` should be here.
The docblock is now also partially incorrect since it says `id` is supposed to be a URL, but in the case of places it's a uuid.

An alternative approach would be passing the url of the event / place and doing a string replacement of eg. `/place/` with `/places/` but then we need to make sure there's a trailing slash to make it work. (Just replacing `place` with `places` could in theory end up as `placess`.)

Something like this could work maybe?
```
var historyUrl = id;
if (historyUrl[historyUrl.length - 1] !== '/') {
    historyUrl += '/';
}
historyUrl += 'history';
historyUrl = historyUrl.replace('/place/', '/places/');

return $http
  .get(historyUrl, defaultApiConfig)
  .then(returnUnwrappedData);
```
Note: I didn't test this, it's just an example

Reply:
@bertramakers I prefer the first approach